### PR TITLE
fix be build error

### PR DESCRIPTION
**Pull request by uestctxq**
bugfix  error: incomplete result type '__float128

### CODE REVIEW

File: be/CMakeLists.txt [hunk 1]
```diff
@@ -359,11 +359,11 @@ endif()
...
 set(CLANG_IR_CXX_FLAGS "-gcc-toolchain" ${GCC_HOME})
-set(CLANG_IR_CXX_FLAGS ${CLANG_IR_CXX_FLAGS} "-std=gnu++11" "-c" "-emit-llvm" "-D__STDC_CONSTANT_MACROS" "-D__STDC_FORMAT_MACROS" "-D__STDC_LIMIT_MACROS" "-DIR_COMPILE" "-DNDEBUG" "-DHAVE_INTTYPES_H" "-DHAVE_NETINET_IN_H" "-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG" "-D__GLIBCXX_BITSIZE_INT_N_0=128" "-D__GLIBCXX_TYPE_INT_N_0=__int128" "-U_GLIBCXX_USE_FLOAT128" "-DLLVM_ON_UNIX")
-if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 7.0)
+set(CLANG_IR_CXX_FLAGS ${CLANG_IR_CXX_FLAGS} "-std=c++11" "-c" "-emit-llvm" "-D__STDC_CONSTANT_MACROS" "-D__STDC_FORMAT_MACROS" "-D__STDC_LIMIT_MACROS" "-DIR_COMPILE" "-DNDEBUG" "-DHAVE_INTTYPES_H" "-DHAVE_NETINET_IN_H" "-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG" "-D__GLIBCXX_BITSIZE_INT_N_0=128" "-D__GLIBCXX_TYPE_INT_N_0=__int128" "-U_GLIBCXX_USE_FLOAT128" "-DLLVM_ON_UNIX")
+# if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 7.0)
     # for support float128
-    set(CLANG_IR_CXX_FLAGS ${CLANG_IR_CXX_FLAGS} "-D__STRICT_ANSI__")
-endif()
+    # set(CLANG_IR_CXX_FLAGS ${CLANG_IR_CXX_FLAGS} "-D__STRICT_ANSI__")
+# endif()
```
Comment (imay) [inline be/CMakeLists.txt]: If this `if` block is useless, just remove it.

**Review by imay — CHANGES_REQUESTED** [review]
Thanks for your contribution, just need a little change

File: be/CMakeLists.txt (hunk 1 above)
Comment (uestctxq) [reply be/CMakeLists.txt]: thanks, i have removed it.

**Review by imay — DISMISSED** [review]
LGTM

**Comment by imay** [comment]
@uestctxq 
Hi, we need our commit log to be imperative mood in the subject line. Could you change your commit log like "Fix be build error"?

**Review by imay — APPROVED** [review]
Good job

**Pull request merged**